Validate create pod form

chloeprezelski opened the pull request.

allysmith12 committed:
Fix touching to go to modify event (#75)

chloeprezelski committed:
Merge branch 'style' of https://github.com/zusoomro/cis400 into validate-pod

chloeprezelski committed:
adds validation to pod creation form

allysmith12 commented on mobile/src/pods/CreatePod.tsx:
```diff
@@ -40,16 +41,35 @@ const CreatePod: React.FC<Props> = ({ navigation, route }) => {
+                json.message
+              );
+            } else {
+              console.log("pods creation successful");
```
remove console?

allysmith12 commented on mobile/src/pods/CreatePod.tsx:
```diff
@@ -59,14 +79,21 @@ const CreatePod: React.FC<Props> = ({ navigation, route }) => {
+              <Button
+                onPress={handleSubmit}
+                title="Submit"
+                disabled={!isValid}
```
Oh lol this is smart ...

allysmith12 reviewed (commented):
LGTM!! Very Nice work chlorida

Commit:
Style (#62)

* Updates splash screen

* Syntactic changes, updates style

* Init work on pods

* Moves load pod into redux state

* Adds manage users page

* Adds image to schedule page

* Fixes bug which kept you signed in

In previous versions, a user might be shown the authenticated view (tab
bar, etc) without actually having a valid active user. This commit fixes
this bug and redirects the user to the login screen

* Removes podid from users

* Adds titles to pod screen names

* Move let to const for constant

* Updates seed file to include events, adds script

Adds the "resetDb" script, which will reset the local dev database and
populate it with the database seed

* Updates pods to use redux actions

* Minor changes

- Stop autocaps in login, register
- Fix eslint again

* Adds types, fixes logout

- Fixes a bug in logout where token wasn't deleted
- Adds types for navigators

* Big changes towards pod resolution

- Changes api backend to remove hardcoding
- Adds events to seed
- Adds tests for conflictingEvents function
- Creates frontend for conflicting events

* Add types, minor work

* Adds avatar for current user, style changes, login refactor

- Changes login and register component to be the same for the time being

* Formats error message

* Modify/create event now share component

* Style pod creation flow, update redux state for pod creation

* Creates type for invite

* Adds email to pod events

* Refactors PodHomeScreen component

- Moves api calls which are not connected to the Redux state into
"podService"
- Moves modal component into PodInviteModal
- Fixes bug where accepting a pod invite didn't immediately put you in
the pod (the Redux action needed to be wrapped in a dispatch)

* Update style

allysmith12 committed:
Validate Event Inputs on the Front and Back end (Duplicate #87) (#94)

* Add Event input validation for the front end

* Validate on the backend

* Formatting

* console.log removed

* Empty PR

* Formatting

allysmith12 committed:
Create Popup for Event Conflict (#96)

* Create Popup for Event Conflict

* Formatting

chloeprezelski committed:
Send invite post pod creation (#101)

* adds ability to send invite post pod creation

* cleans up code

chloeprezelski committed:
Merge branch 'main' of https://github.com/zusoomro/cis400 into validate-pod
